source/val/validation_state.h

```diff
@@ -71,6 +71,9 @@ class ValidationState_t {
                                          // and its values to be used without
                                          // requiring any capability
 
+    // Allow using raw access chains through the RawAccessChainsEXT capability
+    bool raw_access_chains = false;
```

Comment:
I think this is unnecessary. If feature is checked for it should be done via `HasCapability(spv::Capability::RawAccessChainsNV)`.